src/util/tempdir.cpp

```diff
@@ -12,6 +12,8 @@ Author: CM Wintersteiger
 #include <windows.h>
 #include <io.h>
 #include <direct.h>
+#else
+#include <vector>
 #endif
 
 #include <cstdlib>
```

Comment:
Is the `cstring` include still required for non-win32?

Reply:
GetTempFileNameA actually appends to the string!

Reply:
I don't understand. What does it append to which string? According to the documentation it will, given the current code, produce `lpPathBuffer\TLO<some digits>.TMP`.

Reply:
Switched to vector on Windows as well.